### DISCUSSION TITLE
what's vue2 for in frontend code?

### DISCUSSION THREAD

**Discussion by laoshaw** [home-assistant/frontend#21858] [Dashboard] [answered]
trying to understand frontend code, it's lit based but I saw vue2 there, what's vue2 for? Thanks.

**Comment by Misiu** [comment, marked as answer]
https://github.com/home-assistant/frontend/blob/dev/package.json#L141
I think it's only used in https://github.com/home-assistant/frontend/blob/a09f44dcd2b887b3a40116caa4705045dae005ab/src/components/date-range-picker.ts#L4

Not sure if it is used in other places, but if no, then maybe it can be replaced with https://easepick.com/ or with https://litepicker.com/

**Reply by silamon** [reply]
Since the datepicker is written in/for vue, the package vue is also needed to embed it even though we use lit as a framework in general.
